### ISSUE TITLE
[Bug]Trend alias display error when use opentsdb datasource wih metirc like "cpu.$host"

### ISSUE THREAD

**Issue by xiaom0-0** [grafana/grafana#4976]
Grafana version: all
datasource :opentsdb
OS:linux all

when i set a metric name like  "cpu.$host" in templet dashboard,it can get the data of the metirc,bug
the trend name displayed error. it always displayed the first trend's alias.

this my patch of opentsdb datasource

```
source\grafana-2.6.0\public\app\plugins\datasource\opentsdb\datasource.js
296c296,297
<           return target.metric === metricData.metric &&
---
>         	var metric = templateSrv.replace(target.metric,options.scopedVars);
>           return metric === metricData.metric &&
```

**Comment by utkarshcmu** [comment]
Dupe of #1876 , PR submitted in #4910 , Waiting for review/merge. :)

**Comment by xiaom0-0** [comment]
I don't think the patch can fix my bug,please recheck?

**Comment by utkarshcmu** [comment]
@xiaom0-0 You need to upgrade Opentsdb to 2.3 version and then the patch in #4910 will work.

**Comment by xiaom0-0** [comment]
ok,thank you . May be you don't want fix it with opentsdb2.2 or before.

**Comment by torkelo** [comment]
@xiaom0-0 it can't be fixed as the fix depends on feature in 2.3
